guides/water.md

```diff
@@ -8,12 +8,57 @@ nav_order: 4
 # Intro to Water
 ## The Importance of Water
 
-Water is the major component of espresso and acts as a solvent during the extraction phase. Relatively small changes in the general hardness and alkalinity of water can lead to drastic changes in the flavor profile of the resulting espresso. In addition, some municipal water sources, or even bottled water, can form scale when repeatedly heated and cooled inside your espresso machine. This can sometimes clog components, or negatively alter the flow of water in your shiny, spiffy coffee machine. To combat this, the solution is to use soft water. However, water with absolutely no mineral content has been found to negatively affect coffee extraction. Additionally, distilled water can also have a mildly corrosive affect on metal components. So, time to strike some sort of a balance. Time to explore... 
+Water is the main ingredient in coffee. Using good, clean water is important for making good coffee.
+When considering which water to use, there are two main considerations:
+1. Safety - this refers to boiler safety and scale buildup, mainly for espresso.
+2. Taste - the composition of minerals in water can affect the taste of the final drink and may vary by preparation.
 
-## How is the water I am using?
-Many municipal water supplies post regular water reports which can be found by googling "town name water report". The key factors to look for are general hardness (GH) and alkalinity (KH). In the United States, this is often reported as parts per million (aka mg/L of CaCO<sub>3</sub> equivalent). Electronic water TDS (total dissolved solids) meters that measure the conductance of water can also be used to get a general level of TDS in ppm, however note that these meters are not able to estimate GH or KH separetly. An electronic TDS meter is also useful for testing that DI is actually 0 TDS.
 
-## ...The Simple Water Recipe
+## The Basics - Water Safety
+When we refer to water safety, the focus is not on safety for consumption, though obviously you should always make sure that the water you are using is safe to drink.
+For coffee making, and particularly for espresso machines, safety refers to the hardness of the water that can cause scale buildup inside espresso machines and boilers. Hard water can also cause scale buildup in kettles or automatic brewers.
+
+Water hardness is a measure of the concentration of calcium and magnesium in the water. As water is heated and cooled, these minerals will deposit inside any surface it comes into contact with (pipes, boilers, kettles) and can eventually clog or damage these if not properly treated.
+Water hardness is typically measured in ppm (parts per million) of calcium and magnesium.
+
+<ins>As a general guideline, aim for ~50ppm hardness in your water, with >100ppm considered hard and not recommended for use in espresso machines<ins>
```

Comment:
Looks like the `<ins>` sections added in this PR used `<ins>` instead of `</ins>` at the end.